base/examples/test_mne_report.py

```diff
@@ -0,0 +1,35 @@
+# modified from 70_report.py
```

Comment:
I think we should remove this example as it isn't used anywhere. Instead we can add tests in the next Pr, which is already open. And update the readme to explain to users how to mount a volume and point to their own scripts or notebooks.